packages/boxed-expression-component/src/selection/BeeTableSelectionContext.tsx

```diff
@@ -811,37 +827,78 @@ export function BeeTableSelectionContextProvider({ children }: React.PropsWithCh
     const active = selection.active;
 
     const { startRow, endRow, startColumn, endColumn } = getSelectionIterationBoundaries(selection);
+
+    const maxCellCount = currentRefs.get(0)?.size ?? 0;
```

Comment:
I assume that data cells do not support merged view mode. That is why I am using index `0` to get maximum amount of cells in row.